integreat_compass/cms/models/offers/document.py

```diff
@@ -0,0 +1,88 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.core.files.storage import FileSystemStorage
+from django.db import models
+from django.template.defaultfilters import filesizeformat
+from django.utils.functional import cached_property
+from django.utils.translation import gettext_lazy as _
+
+from ...constants import allowed_media
+from ..abstract_base_model import AbstractBaseModel
+from .offer_version import OfferVersion
+
+
+def file_size_limit(value):
+    """
+    This function checks if the uploaded file exceeds the file size limit
+
+    :param value: the size of upload file
+    :type value: int
+
+    :raises ~django.core.exceptions.ValidationError: when the file size exceeds the size given in the settings.
+
+    """
+    if value.size > settings.MEDIA_MAX_UPLOAD_SIZE:
+        raise ValidationError(
+            _("File too large. Size should not exceed {}.").format(
+                filesizeformat(settings.MEDIA_MAX_UPLOAD_SIZE)
+            )
+        )
+
+
+class Document(AbstractBaseModel):
+    """
+    Data model representing a Language.
+    """
+
+    offer_version = models.ForeignKey(
+        OfferVersion, on_delete=models.CASCADE, null=False
+    )
+    name = models.CharField(max_length=255, null=False)
+    blob = models.FileField(
```

Comment:
```suggestion
    file = models.FileField(
```
A bit nit picky, but `blob` usually refers to binary data saved in the database, not on the file system, right? :sweat_smile: 
In this case, the db field only contains a string with the path where the actual file is stored on the disk.

Reply:
Yes, true. I took the name from the ERD, but agree file is clearer